src/Microsoft.Data.SqlClient/src/Microsoft/Data/Common/DbBatchCommandCollection.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Microsoft.Data.Common
+{
+    /// <include file='../../../../doc/snippets/Microsoft.Data.Common/DbBatchCommandCollection.xml' path='docs/members[@name="DbBatchCommandCollection"]/DbBatchCommandCollection/*' />
+    public abstract class DbBatchCommandCollection : IList<DbBatchCommand>
```

Comment:
Same as previous comments, I think this becomes internal-only for netfx, right?

Reply:
see above